app/javascript/src/components/Dashboard/Notes/utils.js

```diff
@@ -0,0 +1,6 @@
+import dayjs from "lib/dayjs";
```

Comment:
I will review your PR once you go through https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=How%20to%20extend%20day.js%3F and fix the mistakes.